.travis.yml

```diff
@@ -26,6 +26,6 @@ jdk:
 before_script:
   - unset _JAVA_OPTIONS
 env:
-  - BUILD_CMD="mvn clean verify javadoc:jar"  # main build of unit tests and javadoc
+  - BUILD_CMD="mvn clean verify"  # main build of unit tests
```

Comment:
The building of the javadocs in CI is to prevent people from adding malformed javadocs which are caught by the javadoc tool. This is common enough that I think it's worth having CI catch it.